src/include/handle_manager.h

```diff
@@ -6,7 +6,8 @@
 #include "backup_manager.h"
 
 /* The maximum length of database name is 17 in English. */
-#define MAX_DB_NAME_LEN 17
+/* For database name + @ + host name */
+#define MAX_DB_NAME_LEN 512
```

Comment:
How about changing it to 256 ? 
`CUB_MAXHOSTNAMELEN` is already used in cubrid as the maximum length of "dbname@hostname"

Reply:
Generally, the maximum hostname is 256. CUB_MAXHOSTNAMELEN also just considers the host name. So, I use 512 iso 256.